searchPage/search.css

```diff
@@ -1,184 +1,277 @@
-@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,900;1,700&display=swap");
-
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400&display=swap");
 * {
-  padding: 0;
-  margin: 0;
-  font-family: "Montserrat", sans-serif;
-  box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+    font-family: "Montserrat", sans-serif;
+    box-sizing: border-box;
 }
+
 header {
-  display: flex;
-  justify-content: center;
+    display: flex;
+    justify-content: center;
 }
+
 .header {
-  width: 90%;
-  display: flex;
-  justify-content: space-between;
-  margin-top: 3.7vh;
+    width: 90%;
+    display: flex;
+    justify-content: space-between;
+    margin-top: 3.7vh;
 }
+
 #logo {
```

Comment:
You already add font-family: 'Roboto', sans-serif; in university but didn't add it for other pages (for logo class)